manifests/init.pp

```diff
@@ -118,7 +118,7 @@
   Boolean                    $enable          = true,
   Boolean                    $manage_repos    = false,
   Boolean                    $manage_packages = true,
-  Boolean                    $manage_selinux  = false,
+  Boolean                    $manage_selinux  = true,
```

Comment:
This switch only makes sense in my opinion if it defaults to false

Reply:
Why? What is the difference between deliberately switching selinux on or off?
By default it's on if selinux is enabled. On the other hand if you wanna disable selinux support if it's on, set manage_selinux to false.